cli/cmd/install.go

```diff
@@ -422,16 +428,6 @@ func renderOverrides(values chartutil.Values, stringData bool) ([]byte, error) {
 	delete(values, "partials")
 	delete(values, "stage")
 
-	// Get Namespace from values
-	valuesTree, err := tree.MarshalToTree(values)
-	if err != nil {
-		return nil, err
-	}
-	namespace, err := valuesTree.GetString("namespace")
```

Comment:
Hmm, This is removed because. Now we don't need to find if `namespace` has been overriden through helm values, etc, and there's now only  a single way to override namespace which is through `--namespace` flag propogated into `controlPlaneNamespace`, right?

Reply:
That's correct.